core/src/main/java/bio/terra/pearl/core/model/workflow/ParticipantTask.java

```diff
@@ -26,9 +26,11 @@ public class ParticipantTask extends BaseEntity {
     private boolean blocksHub; // whether this task blocks the participant from doing optional tasks in the hub
     private UUID studyEnvironmentId;
     private UUID enrolleeId;
+    private UUID assignedAdminUserId;
     private UUID portalParticipantUserId;
     private UUID surveyResponseId;
     private UUID kitRequestId;
+    private UUID participantNoteId;
 }
```

Comment:
The dispositionNote field was removed in this refactor. I didn't see any usages of it in production when I last checked. Should I retain it?